pcre/plan.sh

```diff
@@ -2,23 +2,27 @@ pkg_name=pcre
 pkg_origin=core
 pkg_version=8.38
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="A library that implements Perl 5-style regular expressions"
+pkg_upstream_url="http://www.pcre.org/"
 pkg_license=('bsd')
-pkg_source=http://ftp.csx.cam.ac.uk/pub/software/programming/${pkg_name}/${pkg_name}-${pkg_version}.tar.bz2
+pkg_source="http://ftp.csx.cam.ac.uk/pub/software/programming/${pkg_name}/${pkg_name}-${pkg_version}.tar.bz2"
 pkg_shasum=b9e02d36e23024d6c02a2e5b25204b3a4fa6ade43e0a5f869f254f49535079df
 pkg_deps=(core/glibc core/gcc-libs)
 pkg_build_deps=(core/coreutils core/diffutils core/patch core/make core/gcc)
 pkg_bin_dirs=(bin)
 pkg_include_dirs=(include)
 pkg_lib_dirs=(lib)
+pkg_pconfig_dirs=(lib/pkgconfig)
```

Comment:
For whatever reason, `hab` is not updating my `$PKG_CONFIG_PATH` when I use `pcre` as a dependency.  I had to explicitly set this parameter otherwise I have to be more verbose in my plans.